conf/nginx.conf.server.template

```diff
@@ -18,41 +18,12 @@ resolver ${NAMESERVER} ipv6=off valid=10s;
 
 # ====================== Expiration Rules ====================== #
 
-## Configures expiration for upstream content based on content type.
-map $upstream_http_content_type $content_type_expiry {
-    default                                 5s;
-    'application/json'                      5s;
-    'application/pdf'                       max;            # max means 10 years to nginx
-    'application/octet-stream'              max;
-    'binary/octet-stream'                   max;
-    'image/jpeg'                            max;
-    'image/gif'                             max;
-    'image/png'                             max;
-    'image/bmp'                             max;
-    'image/webp'                            max;
-    'video/webm'                            max;
-    'video/ogg'                             max;
-    'video/mp4'                             max;
-    'application/binary'                    -1s;            # used by HLS streams
-    '~*mpegurl'                             -1s;
-}
-
-map $host $host_expiry {
-    default                     $content_type_expiry;
-    # foo.com                   30d;                        # example of expiry by host
-    "~*uplynk.com"              -1s;                        # known HLS stream provider
-}
```

Comment:
These settings were never actually used because we were setting DISABLE_FORWARD_CACHE_EXPIRY_OVERRIDE=1, which removes the `expires $cache_expiry;` part of the config. I've removed them to avoid confusion in the future.